mParticle-ComScore/MPKitComScore.m

```diff
@@ -148,14 +161,14 @@ - (instancetype)initWithConfiguration:(NSDictionary *)configuration startImmedia
 }
 
 - (MPKitExecStatus *)beginSession {
-    [CSComScore onUxActive];
+    [SCORAnalytics notifyUxActive];
```

Comment:
Are sessions and UI/UX usage equated by comScore? Would this be the case to invoke on `didBecomeActive` and `didEnterBackground`?